sdk/communication/communication-identity/test/public/communicationIdentityClient.spec.ts

```diff
@@ -39,29 +41,25 @@ matrix([[true, false]], async function (useAad: boolean) {
       assert.isString(user.communicationUserId);
     });
 
-    it("successfully creates a user and token", async function () {
-      const { user: newUser, token } = await client.createUserAndToken(["voip"]);
+    given([
+      { scopes: chatScope, description: "chat scope" },
+      { scopes: voipScope, description: "voip scope" },
+      { scopes: multipleScopes, description: "multiple scopes" },
+    ]).it("successfully creates a user and token", async function (input) {
+      const { user: newUser, token, expiresOn } = await client.createUserAndToken(input.scopes);
+      assert.isTrue(isCommunicationUserIdentifier(newUser));
       assert.isString(newUser.communicationUserId);
       assert.isString(token);
-    });
-
-    it("successfully gets a token for a user [single scope]", async function () {
-      const user: CommunicationUserIdentifier = await client.createUser();
-      const { token, expiresOn } = await client.getToken(user, ["chat"]);
-      assert.isString(token);
       assert.instanceOf(expiresOn, Date);
     });
 
-    it("successfully gets a token for a user [multiple scopes]", async function () {
+    given([
+      { scopes: chatScope, description: "chat scope" },
```

Comment:
as we are using the test arguments only in this file, I would put the arguments directly here instead of creating const values in the `utils`, as it makes it more readable what kind of args are tested, same for the test args of the line 44

Reply:
Adding arguments directly was giving error, since we need to pass TokenScope[] object and it's a type. So we need to create const and then pass it as an argument. If we want, I can create const in the same test file.

Reply:
It's really weird that we did not get the same error for:
`const { token, expiresOn } = await client.getToken(user, ["chat", "voip"]);`
anyway, if passing directly is an issue for us, then I'd prefer putting the const in this file(after line 23) and reusing the const in the tests instead of creating new scopes